### PR TITLE
Add proposal counting per schema

### LINKED ISSUE #326: keep counts of proposals
We are doing a lot of separate queries to count proposals, and we coudl avoid that if these counters are part of the index. So I'd like to add the folllowing fields:

``` 
Scheme.numberOfQueuedProposals
Scheme.numberOfPreBoostedProposals
Scheme.numberOfBoostedProposals
DAO.numberOfQueuedProposals
DAO.numberOfPreBoostedProposals
DAO.numberOfBoostedProposals
```
For the other states (`ExpiredInQueue`, `Executed`, `QuietEndingPeriod`) we are not doing separate queries, but it could be useful to have these counts available anyway to help us with paging and showing totals.

### PR DESCRIPTION
**Pull request by ben-kaufman**
Fix #326

### CODE REVIEW

File: src/mappings/Controller/schema.graphql [hunk 1]
```diff
@@ -23,7 +23,9 @@ type ControllerScheme @entity {
 	schemeRegistrarParams: SchemeRegistrarParam
 	uGenericSchemeParams: UGenericSchemeParam
 	genericSchemeParams: GenericSchemeParam
-
+	numberOfQueuedProposals: BigInt!
```
Comment (orenyodfat) [inline src/mappings/Controller/schema.graphql]: is that not suppose to be per dao ?

Comment (jellegerbrandy) [reply src/mappings/Controller/schema.graphql]: yes, indeed it is, good call.

It is probably better to count and store this ont he `gpqueue` object.

**Comment by orenyodfat** [comment]
Though gpque is per scheme (not dao) as well . Though if this what’s needed
so ..let it be

**Comment by ben-kaufman** [comment]
I don't see the issue, the scheme is per DAO.

**Pull request merged**